src/scripts/slurm_submit.sh

```diff
@@ -85,8 +85,17 @@ bls_add_job_wrapper
 ###############################################################
 
 datenow=`date +%Y%m%d`
-jobID=`${slurm_binpath}/sbatch $bls_tmp_file` # actual submission
-retcode=$?
+retry=0
+MAX_RETRY=3
+until [ $retry -eq $MAX_RETRY ] ; do
```

Comment:
The first attempt is `retry=0`, once `retry=3`, the loop condition fails and breaks. That should be three tries, right?